testing/py3-isodate/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Robert Sacks <robert@sacks.email>
+# Maintainer: Robert Sacks <robert@sacks.email>
+pkgname="py3-isodate"
+_pyname="isodate"
+pkgver="0.6.0"
+pkgrel=0
+pkgdesc="ISO 8601 date/time parser"
+url="https://pypi.org/project/isodate/"
+arch="noarch"
+license="custom"
```

Comment:
I think I faintly recall a discussion about this on IRC, but isn't this BSD-3-Clause?

Reply:
It is missing this last part of BSD-3-Clause:

> (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

Reply:
Oh, alright